agot-bg-game-server/src/common/ingame-game-state/game-data-structure/Game.ts

```diff
@@ -319,7 +321,7 @@ export default class Game {
         return this.starredOrderRestrictions[place];
     }
 
-    serializeToClient(admin: boolean): SerializedGame {
+    serializeToClient(admin: boolean, knowsNextWildlingCard: boolean): SerializedGame {
```

Comment:
It's really not the prettiest way of doing it, but I guess it's the only way to do it. 👍

Reply:
I experimented with sending in the `Player` or `House` objects, but it required null sanitation  in `Game` which kinda made it even less nice, but would _probably_ be a better idea long-term.